optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/UseCasesAndExamples-chapter.adoc

```diff
@@ -11,9 +11,17 @@
 [[examplesOverview]]
 == Examples Overview
 
-Planner has several examples.
-In this manual we explain mainly using the n queens example and cloud balancing example.
-So it's advisable to read at least those sections.
+Planner has several examples. In this manual we explain mainly using the _n_ queens example and cloud balancing example. So it is advisable to read at least those sections.
```

Comment:
I agree that we should have a consistent example used throughout where possible. If a particular concept is better explained with n queens, then obviously we should use the example that best suits, but where we can use cloud balancing instead we should for the sake of consistency. It is much easier to understand a concept when it's building on a familiar example.